public/app/features/scopes/selector/ScopesSelectorService.ts

```diff
@@ -46,6 +49,12 @@ export interface ScopesSelectorServiceState {
 }
 
 export class ScopesSelectorService extends ScopesServiceBase<ScopesSelectorServiceState> {
+  private interactionProfiler = new behaviors.SceneInteractionProfiler({
```

Comment:
This is awkward to use scene's behavior in non-scenes object. This makes me think the interaction profiler should be a class for non-scenes use cases, while a behavior should wrap it for scenes case.